airbyte-cdk/python/airbyte_cdk/sources/streams/http/error_handlers/json_error_message_parser.py

```diff
@@ -45,4 +45,7 @@ def parse_response_error_message(self, response: requests.Response) -> Optional[
             body = response.json()
             return self._try_get_error(body)
         except requests.exceptions.JSONDecodeError:
-            return None
+            try:
+                return response.content.decode("utf-8")
+            except Exception:
+                return None
```

Comment:
This change allows this method to also properly handle response bodies which are just string values, since that currently causes a JSONDecodeError